<ui version="4.0" >
 <class>AccountWindow</class>
 <widget class="QDialog" name="AccountWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add/Update account</string>
  </property>
  <widget class="QLineEdit" name="txtName" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>30</y>
     <width>113</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbName" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>10</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPassword" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>10</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Password</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QPushButton" name="pbOk" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbCancel" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
